README.md

```diff
@@ -399,7 +399,7 @@ To roll back, do the following
 
   <details>
 
-  <summary>Azure Firewall disassociate PIP</summary>
+  <summary>AzFw disassociate pip</summary>
```

Comment:
leave this as `Azure Firewall` lets not assume everyone knows what `AzFw` means 😜

Reply:
Changed abbrievation to full name